### PR TITLE
Create Microsoft.DataTools.IntegrationServices.yaml

### PR DESCRIPTION
**Pull request by philipwolfe**
Adding Visual Studio 2019 SSIS Project installer.

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24488)

### CODE REVIEW

File: manifests/m/Microsoft/DataTools/IntegrationServices/3.14/Microsoft.DataTools.IntegrationServices.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Publisher: Microsoft Corporation
+PackageIdentifier: Microsoft.DataTools.IntegrationServices
+PackageVersion: 3.14
+License: Copyright (c) Microsoft Corporation. All rights reserved.
+ShortDescription: This project may be used for building high performance data integration and workflow solutions, including extraction, transformation, and loading (ETL) operations for data warehousing.
+PackageUrl: https://marketplace.visualstudio.com/items?itemName=SSIS.SqlServerIntegrationServicesProjects
+Tags: 
+- "dtproj"
+- "Visual Studio"
+- "SSIS"
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/DataTools/IntegrationServices/3.14/Microsoft.DataTools.IntegrationServices.yaml:11]:
```suggestion
- dtproj
- visual-studio
- ssis
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
**Please make sure to test your WinGet package inside of a virtual machine**:
image

File: manifests/m/Microsoft/DataTools/IntegrationServices/3.14/Microsoft.DataTools.IntegrationServices.yaml (hunk 1 above)
Comment (philipwolfe) [reply manifests/m/Microsoft/DataTools/IntegrationServices/3.14/Microsoft.DataTools.IntegrationServices.yaml:11]: @ItzLevvie Did your virtual machine have Visual Studio 2019 installed?  This is an extension for VS that is packaged as an EXE.

Comment (ItzLevvie) [reply manifests/m/Microsoft/DataTools/IntegrationServices/3.14/Microsoft.DataTools.IntegrationServices.yaml:11]: cc @philipwolfe: No, all manifests are validated on a clean installation of Windows with no dependencies installed.

This package has an install-time dependency, so it looks like you'll have to wait for dependency support to be added which is being tracked at https://github.com/microsoft/winget-cli/issues/163.

So please add your upvote there.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please read https://github.com/microsoft/winget-pkgs/pull/24488#discussion_r688630008.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You can put this PR in DRAFT mode for now until dependency support is added.

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**